code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -374,15 +374,15 @@ var/veil_thickness = CULT_PROLOGUE
 /obj/item/proc/get_cult_power()
 	return 0
 
+var/static/list/valid_cultpower_slots = list(
```

Comment:
Minor incomprehension : 
I don't see why that should we a global instead of, say a static list of the bloodcult fac.
I don't believe global variables care if they are static or not.

Reply:
Yes, `static` and `global` mean the same thing here: nothing.

Reply:
@ShiftyRail I'd rather make it a static var of the /datum/rune_spell type then, so I don't have to look up the faction.